extensions/EXT_disjoint_timer_query/extension.xml

```diff
@@ -224,25 +224,34 @@ interface EXT_disjoint_timer_query {
         ext.endQueryEXT(ext.TIME_ELAPSED_EXT);
 
         // ...at some point in the future, after returning control to the browser and being called again:
```

Comment:
ah, I wasn't clear that this code would be called multiple times - perhaps just mention that in this comment

Reply:
Good idea. Done throughout.